crawl-ref/source/dat/descript/zh/unident.txt

```diff
@@ -7,23 +7,28 @@ book
 
 一本晦涩难懂的古书，写着不知何源的魔咒。
 %%%%
+deck of cards
+
+一副充满强大魔力的卡牌，牌面上用恶魔的脓血画着各种各样的图案。
+抽牌后，可能会对你有利，但也可能将你置于险境之中。
```

Comment:
Hm, is it possible to encounter an unidentified deck of cards these days..? Surely not, right?

Well, no harm, I guess.